### DISCUSSION TITLE
how can I do scheduling using metadata ? Actually I want to Schedule multiple reports on different time ,will it Possible to do in Superset ?

### DISCUSSION THREAD

**Discussion by VivekGupta97** [apache/superset#17857] [Q&A / Help]

**Comment by request-info** [comment]
We would appreciate it if you could provide us with more info about this issue/pr! Please do not leave the `title` or `description` empty.

**Comment by VivekGupta97** [comment]
Is there possible to schedule reports on different time with different user

**Comment by VivekGupta97** [comment]
Basically I want scheduling email using metadata

**Comment by rusackas** [comment]
@betodealmeida @eschutho may have some insight here, but for the time being, since this is a how-to question rather than an Issue, I'm converting it to a discussion, and folks can hopefully dive into it there.

**Comment by rusackas** [comment]
You may also want to reach out to the Slack community here as well... let me know if you need any pointers on connecting there (it's in the docs)

**Reply by VivekGupta97** [reply]
I posted there as well but my Query is not resolved yet  could you Please help me out

**Comment by eschutho** [comment]
Hi @VivekGupta97 You can schedule each report separately for a different time. Are you looking to have one report send out different times based upon who the email is sent to?

**Reply by VivekGupta97** [reply]
I want to send one report at the different time to other user so is it possible to do so ?

**Comment by VivekGupta97** [comment]
Actually I want to Send Report on Different Time . If there are 500 reports and I want to send them to the user on different time so will it be possible or just I need to create different report schedule for that manually ?

**Reply by betodealmeida** [reply]
One report can be scheduled at a specific frequency. For example you can have one report being sent every 5 minutes, or every hour, or every Monday.

Each report goes to a specific group of users.

If you want to send a report to two different people at different times, then you need to create two different reports.
